neofetch

```diff
@@ -53,7 +53,7 @@ get_distro() {
     [[ "$distro" ]] && return
 
     case "$os" in
-        "Linux")
+        "Linux" | "BSD" | "MINIX")
             if [[ "$(< /proc/version)" == *"Microsoft"* || "$(< /proc/sys/kernel/osrelease)" == *"Microsoft"* ]]; then
```

Comment:
Now that we've added `2>/dev/null` to `get_distro` we can remove all traces of `2>/dev/null` from inside the function.